.github/libs/GithubUtils.ts

```diff
@@ -459,6 +463,7 @@ class GithubUtils {
      */
     static getLatestWorkflowRunID(workflow: string | number): Promise<number> {
         console.log(`Fetching New Expensify workflow runs for ${workflow}...`);
+        // eslint-disable-next-line rulesdir/no-default-id-values
```

Comment:
I don't think [this](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#default-value-for-inexistent-ids) style rule should apply to a github issue automation. Let me know if you think otherwise.

Reply:
Yea, that seems fine.